packages/venia-concept/src/ServiceWorker/setupWorkbox.js

```diff
@@ -29,7 +29,7 @@ export default function() {
     clientsClaim();
```

Comment:
Doesn't have to be in this scope, but I'd like to audit the purpose of this file. `skipWaiting` is going to be removed in the future release, and from my quick review of docs, I'm not sure this is needed since we're using the [webpack plugin](https://developers.google.com/web/tools/workbox/reference-docs/latest/module-workbox-webpack-plugin.GenerateSW#GenerateSW), which has flags for both `skipWaiting` and `clientsClaim`.

I _almost_ missed bumping this script version too, seems fragile to have this hidden dependency in here. The webpack plugin also seems to have an option for `importScripts`, so this file might not be needed anymore if this can all be accomplished at buildtime with the plugin.

Reply:
Would you mind creating a ticket for this so we don't forget?

Reply:
For sure. I was hoping to bring this up to @sirugh and @revanth0212 and see how they felt about it. If it seems simple and straight forward, might make sense to do now so we don't have to fully regress SW functionality twice.

I'll create the ticket now, and we can just close it if we include it here.